package.json

```diff
@@ -11,9 +11,10 @@
     "update-contributors": "all-contributors generate",
     "commit": "git-cz",
     "validate": "npm-run-all --parallel lint cover --sequential check-coverage",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post",
     "check-coverage": "nyc check-coverage --statements 100 --branches 100 --functions 100 --lines 100",
-    "report-coverage": "cat ./coverage/lcov.info | node_modules/.bin/codecov"
+    "report-coverage": "npm run travis-after-all && cat ./coverage/lcov.info | node_modules/.bin/codecov",
+    "semantic-release": "npm run travis-after-all && semantic-release pre && npm publish && semantic-release post",
+    "travis-after-all": "travis-after-all"
```

Comment:
Wouldn't it be better to do

```JSON
  "travis-after-all": "travis-after-all && npm run report-coverage && npm run semantic-release"
```

in `package.json` and the following in `.travis.yml`?

```YAML
after_success:
  - npm run travis-after-all
```

I know this could be considered nitpicking :stuck_out_tongue_winking_eye: